src/services/github.js

```diff
@@ -28,7 +28,17 @@ export async function publishNewDao(daoAddress, daoMetadata, chainId) {
       token_logo:
         daoMetadata.token_logo &&
         `${ASSETS_FOLDER_BASE}/${daoMetadata.name}/token_logo.${daoMetadata.token_logo.imageExtension}`,
-    })
+    }
+
+    // if case it's a Boboli type we set the wrappableToken
+    if (daoMetadata.wrappable_token_logo !== null) {
+      newDaoItem.wrappableToken = {
+        logo: `${ASSETS_FOLDER_BASE}/${daoMetadata.name}/wrappable_token_logo.${daoMetadata.wrappable_token_logo.imageExtension}`,
+      }
+    }
```

Comment:
@kamikazebr we are going to need to include this in the `dao-list-middleware` right?

Reply:
Just on github.ts file we need added it after. @0xGabi